pkgs/development/interpreters/bqn/cbqn/default.nix

```diff
@@ -4,6 +4,7 @@
 , genBytecode ? false
 , bqn-path ? null
 , mbqn-source ? null
+, replxx ? true
```

Comment:
I would call it `enableReplxx` to be more inline with the rest of nixpkgs – i.e. so it is not overloaded with how a `replxx` library would be packaged.

Additionally, it should default to `false`, as it breaks the BQN interaction in `bqn-mode`, for example (I believe).

Reply:
I got it.